RTCError and RTCErrorEvent docs (#44047)
mdn/content: files/en-us/web/api/rtcerror/sctpcausecode/index.md

{{APIRef("WebRTC")}}

The **`sctpCauseCode`** read-only property of the {{domxref("RTCError")}} interface provides the {{Glossary("SCTP")}} cause code explaining why the SCTP negotiation failed, if the `RTCError` represents an SCTP error.

## Value

> An unsigned long integer value specifying SCTP cause code explaining why the error occurred.

Reviewer: ```suggestion
A positive integer specifying the SCTP cause code explaining why the error occurred.
```

Author: THanks. Thought I caught all of these!

> This property is `null` if the error isn't an SCTP error, with its {{domxref("RTCError.errorDetail", "errorDetail")}} property set to `sctp-failure`.

Reviewer: ```suggestion
This property is `null` if the error isn't an SCTP error, indicated by the {{domxref("RTCError.errorDetail", "errorDetail")}} property being set to `sctp-failure`.
```
